### DISCUSSION TITLE
Custom eventContent for event that is created by select?

### DISCUSSION THREAD

**Discussion by shijenwai** [vkurko/calendar#350] [Q&A] [answered]
The currently selected area only displays the start and end times. I want it to show how many hours and minutes are in the selected range. Is this possible?

I have modified eventContent, but it only affects events where the display is set to auto or background.

**Comment by vkurko** [comment, marked as answer]
`eventContent` is fired for all events, including _helper_ ones, which is the select event. You can distinguish this event from others by checking the `display` field, which should be equal to the `"preview"` value, as well as the `id` field, which is equal to `"{select}"`.

**Reply by shijenwai** [reply]
Ah, I understand now. Thank you very much.
I previously restricted event.display when returning the DOM in eventContent, but I overlooked it.
